docs/beaver-themer/developer/customize-field-connections-themer.md

```diff
@@ -56,17 +56,23 @@ The config array accepts the following properties:
  The label for your connection that will show in the menu.
 
 **group (required)**  
-  The group you want your connection to appear in. Currently, the following groups are built in:
-
-  * general
-  * comments
-  * author
-  * user
-  * site
-  * advanced
-  * woocommerce
-
-See the section below to add custom groups.
+  The group you want your connection to appear in. Currently, the following groups are built in (array key values in brackets):
```

Comment:
Sorry I know you call these brackets but most of the KB is in US English so these would be 'parentheses'.